backend/src/jobs.rs

```diff
@@ -1233,6 +1233,25 @@ pub enum JobPayload {
     RawFlow { value: FlowValue, path: Option<String> },
 }
 
+lazy_static::lazy_static! {
+    // TODO: these aren't synced, they should be moved into the queue abstraction once/if that happens.
+    static ref JOBS_PUSHED: prometheus::IntCounter = prometheus::register_int_counter!(
+        "jobs_pushed",
```

Comment:
let's prefix all the server metrics with `server.` WDYT ?

Reply:
sounds good, but I think the job/queue specific metrics should have their own category (as that logic is shared / is another component). maybe `queue_` instead of `jobs_`?